### ISSUE TITLE
[Feature Request] Load proper config when the VSH/XMB is loaded

### ISSUE THREAD

**Issue by ultra0000** [RPCS3/rpcs3#11888]
## Quick summary
I think it would be a good idea to have the emulator automatically load the proper config whenever the VSH/XMB is loaded.

## Details
Basically just have the emulator load the configuration from https://wiki.rpcs3.net/index.php?title=Help:Frequently_Asked_Questions#Configuration only when the XMB/VSH is loaded (then whenever the user's done with it, revert back to the previous config).

#### 1. Please describe, what part of RPCS3 would be affected by your feature:
* UI

#### 2. Please tell us, why your feature is important to RPCS3.
It would make it easier to load the VSH/XMB, since the user wouldn't have to manually change the config every time they want to load it and then revert it when they're done with it.

#### 3. Please attach screenshots of the feature implemented in other projects.
Dolphin has a feature where if you load a game, it loads the proper configuration (including the Wii Menu). However i feel like this could be a bit too complex for RPCS3, so unless you'd actually want to implement it, I'd say keep it for the VSH/XMB.
It highlights the changes in bold text.
![image](https://user-images.githubusercontent.com/43279290/165597655-d29ae9c4-7b2a-415d-8af7-d1156c71ab14.png)

#### 4. Please provide your system configuration:
* OS: Windows 11
* CPU: AMD A8 7600
* GPU: Radeon R7 Graphics (iGPU)
* Driver version: 21.5.2
* RAM: 8 GB
(It is a mediocre PC however it's able to run a few games fine, including the VSH/XMB)

**Comment by Darkhost1999** [comment]
VSH/XMB works with default settings just fine aside from the "broken" waves. Manually setting configs for all .self and .elf files is a good idea though. Allowing custom configs for all bootable files will be a nice improvement.

**Comment by dio-gh** [comment]
I don't see why scraping the settings for VSH would be necessary, when they could be hardcoded in. It's a special case, the settings won't change for a long time, and compared to just hard coding it this would be significantly more elaborate.

**Comment by Asinin3** [comment]
Yeah honestly they should just be hardcoded into RPCS3. If RSXAudio's only functionality is for VSH and nothing else, then I think it should just be removed from the UI entirely and hardcoded in for VSH. There's no functionality gained by having it as a setting otherwise and it only makes our settings even more intimidating.

But if other apps can use it then by all means keep it as a setting.

**Comment by Megamouse** [comment]
I think this issue is kinda redundant if someone ever adds the option to boot games with "recommended settings"

**Comment by dio-gh** [comment]
That would take a little more than just the adding the option client side...

**Comment by Darkhost1999** [comment]
We now can have custom configs with vsh so as everyone can have their own definition of proper.
I consider this issue fixed. What's the agreed upon decision here?

**Comment by Satan86** [comment]
The issue is primarily about loading settings automatically, which is already covered in a differente Git Hub issue, #4434. Once the emulador as a whole gets the feature, the XMB will be covered as well.

**Comment by ultra0000** [comment]
I suppose I should close this, then?

**Comment by dio-gh** [comment]
I don't think you should close it yourself, no. The maintainers need to decide what to do, these are not actually the same demands, even if they overlap.

> the XMB will be covered as well.

Only if that is approved, which is of no guarantee. There's only one VSH, it can make sense to treat it in a special way, which is what this ticket is partially about, independent of whether there will ever be a generic GameDB implemented.
